tests/jballerina-integration-test/src/test/resources/packaging/module_execute_invocation_project/main.bal

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
```

Comment:
I see we have used `ballerina/io` in other integration tests like ModuleExecutionFlowTests also. Should we avoid that?

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/39588